README.md

```diff
@@ -65,6 +65,41 @@ One.sendInteraction("/interactionPath", { 'key' : 'value' });
 ```
 * See example of usage [here](https://github.com/thunderheadone/one-sdk-flutter/tree/master/examples/flutter_example/lib/main.dart#L128)
 
+To send an Interaction request and retrieve the response, call the following method:
+```dart
+One.sendInteraction("/interactionPath").then((response) {
+    print('Interaction response tid = ${response[oneResponseTidKey]}');
+    print('Interaction response Interaction path = ${response[oneResponseInteractionPathKey]}');
+    print('Interaction response optimization points = ${response[oneResponseOptimizationPointsKey]}');
+});
+```
+
+### Send a response code
+To send a response code, call the following method:
+```dart
+One.sendResponseCode("code", "/interactionPath");
+```
+
+### Opt an end-user out of tracking
+
+To opt an end-user out of all tracking options, when the end-user does not give permission to be tracked in the client app, call the following method:
+```dart
+// Opts out of all tracking options.
+One.optOut(true);
+// or
+One.optOut(true, [OneOptOptions.allTracking]);
+```
+
+To opt out of specific tracking options, you can pass an array of `OneOptOptions` parameters:
+```dart
+// Use this option to opt an end-user out or in of all city/country level tracking.
+One.optOut(true, [OneOptOptions.cityCountryDetection]);
+// iOS platform specific option to opt out of keychain storage.
+One.optOut(true, [OneOptOptions.iOS_keychainTidStorage]);
+// iOS platform specific option to opt out of pasteboard tid storage.
+One.optOut(true, [OneOptOptions.iOS_pasteboardTidStorage]);
+```
```

Comment:
Should we mention how to opt back in? Replacing `true` by `false`? 
Because just deleting that line does not opt me back in. Might be confusing for the users.

Reply:
Added opt in instructions